globals.go

```diff
@@ -38,6 +38,10 @@ const (
 var (
 	globalQuiet = false // Quiet flag set via command line
 	globalTrace = false // Trace flag set via environment setting.
+
+	// By default root is disallowed.
+	globalRootDisallow = true
+
```

Comment:
global variables are useful to save states for the life of a program. In this case, this would mean that certain functions that are not allowed in root-mode will alone be disabled. Simpler approach is to simply check at the right place and exit.